onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -718,72 +728,102 @@ std::unique_ptr<IndexedSubGraph> MIGraphXExecutionProvider::GetSubGraph(const st
   int input_order = 0;
```

Comment:
Please move local variables `erased`, `output_order`, and `input_order` to the inner scope of the `is_skip_fuse` conditional block, as those are not used outside the block.